tests/test_distributions.py

```diff
@@ -302,6 +305,18 @@ def test_expand_margins(self, flat_array):
         assert x1 == x2
         assert y1 + height * 2 == pytest.approx(y2)
 
+    def test_multiple_rugs(self, long_df):
+
+        values = np.linspace(start=0, stop=1, num=5)
+        ax = lineplot(x=values, y=values)
```

Comment:
Having the `lineplot` helps demonstrate the issue visually but it doesn't seem necessary to trigger the error programmatically, so I don't think we need it in the test.

Reply:
Ah I actually had assumed that `lineplot` was required, but I see you're correct. I've made the change 👍